packages/mjml-parser-xml/src/index.js

```diff
@@ -74,6 +74,9 @@ export default function MJMLParser(xml, options = {}, includedIn = []) {
         tagName: 'mj-raw',
         content: `<!-- mj-include fails to read file : ${file} at ${partialPath} -->`,
         children: [],
```

Comment:
why not `error: { type: 'include',  params: {file, partialPath}}` here

Reply:
yes better indeed to keep this separated from attributes